docs/src/routes/docs/components/typography.svx

```diff
@@ -0,0 +1,89 @@
+---
+name: Typography
+---
+
+<script>
+  import { H1, H2, H3, Headline, Subhead, Label } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+</script>
+
+# Typography
+
+Typography elements with embedded responsive styles.
+
+Also comes with a `<TypographyStyles>` component, which only adds global styles, setting the `$font` for the whole page, coloring links, etc. A good place to put it for Sapper, for example, would be anywhere in the main [layout file](https://sapper.svelte.dev/docs#Layouts).
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Headline>Main stuff!</Headline>
+    <Subhead>Make sure you don't miss out!</Subhead>
+
+    <H1>Heading 1</H1>
+    <H2>Heading 2</H2>
+    <H3>Heading 3</H3>
+
+    <Label>label</Label>
+    <Label small>smaller label</Label>
+  </div>
+  <div slot="source">
+
+```svelte
+<Headline>Main stuff!</Headline>
+<Subhead>Make sure you don't miss out!</Subhead>
+
+<H1>Heading 1</H1>
+<H2>Heading 2</H2>
+<H3>Heading 3</H3>
+
+<Label>label</Label>
+<Label small>smaller label</Label>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+_This property only applies to `<Label>`_.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`small`** | `false` | `boolean` | Makes the `<Label>` smaller. |
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`...`** |  | `any` | Everything will be passed to the underlying element. |
+
+### Class Properties
+
+_The following only applies to `<Headline>`, `<Subhead>` and `<Label>` components._
```

Comment:
Not a fan of the Oxford comma, I see)

Reply:
Russian upbringing surfaces

Reply:
Technically, both are correct, but I side with the Oxford comma team)